modules/cnf-installing-sriov-fec-operator.adoc

```diff
@@ -0,0 +1,172 @@
+
+[id="installing-the-sriov-fec-operator_{context}"]
+= Installing the SR-IOV FEC Operator
+
+The role of the OpenNESS Operator for Intel Wireless Forward Error Correction (FEC) Accelerator is to orchestrate and manage the devices exposed by a range of Intel vRAN FEC acceleration hardware within the {product-title} cluster.
+
+One of the most compute-intensive 4G/LTE and 5G workloads is RAN layer 1 (L1) forward error correction (FEC), which resolves data transmission errors over unreliable or noisy communication channels. FEC technology detects and corrects a limited number of errors in 4G/LTE or 5G data without the need for retransmission.
+
+The FEC devices are provided by the Intel® FPGA PAC N3000 FPGA and the Intel® vRAN Dedicated Accelerator ACC100 for the vRAN use case.
+
+[NOTE]
+====
+The Intel® FPGA PAC N3000 FPGA requires flashing with an 4G/LTE or 5G bitstream.
+====
+
+
+The SR-IOV FEC Operator provides functionality to create desired virtual functions (VFs) for the FEC device, binds them to appropriate drivers, and configures the VFs queues for desired functionality in 4G/LTE or 5G deployment.
+
+As a cluster administrator, you can install the OpenNESS SRIOV-FEC Operator by using the {product-title} CLI or the web console.
+
+[id="installing-sriov-fec-operator-using-the-cli_{context}"]
+== Installing the SR-IOV FEC Operator by using the CLI
+
+As a cluster administrator, you can install the Operator by using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the SRIOV-FEC Operator by completing the following actions:
+
+.. Define the `vran-acceleration-operators` namespace by creating a file named `sriov-namespace.yaml` as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: vran-acceleration-operators
+    labels:
+       openshift.io/cluster-monitoring: "true"
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-namespace.yaml
+----
+
+. Install the SRIOV-FEC Operator in the namespace you created in the previous step by creating the following objects:
+
+.. Create the following `OperatorGroup` CR and save the YAML in the `sriov-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: vran-operators
+    namespace: vran-acceleration-operators
+spec:
+    targetNamespaces:
+      - vran-acceleration-operators
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-operatorgroup.yaml
+----
+
+
+.. Run the following command to get the `channel` value required for the next step.
++
+[source,terminal]
+----
+$ oc get packagemanifest sriov-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
```

Comment:
@kquinn1204, is the `defaultChannel` always correct? For SR-IOV Network Operator, the `channel` is based on the OCP version, so this is handled a bit differently, because `defaultChannel` is not necessarily correct.

Reply:
here the channel is always stable for the operator

Reply:
Thanks, @SchSeba I was not sure on this one. Hopefully, a QE will bottom out any other technical issues